client/src/components/Game/FabricCanvas.js

```diff
@@ -0,0 +1,74 @@
+import React, { useContext, useEffect } from 'react';
+import { FabricContext } from './FabricContextProvider';
+import { fabric } from 'fabric';
+import { saveAs } from 'file-saver';
+
+const FabricCanvas = props => {
+    const [canvas, initCanvas] = useContext(FabricContext);
+    const canvasX = 1000;
+    const canvasY = 800;
```

Comment:
I'm assuming these are deeply immutable, which if so can be capitalized (unless ur planning a fancy modifiable canvas size or smth).
```suggestion
    const CANVAS_X = 1000;
    const CANVAS_Y = 800;
```